src/v2/Components/ArtworkFilter/Utils/isDefaultFilter.tsx

```diff
@@ -20,6 +21,8 @@ export const isDefaultFilter: (
       name === "artistNationalities" ||
       name === "materialsTerms":
       return value.length === 0
+    case !!defaultValues[name]:
+      return value === defaultValues[name]
```

Comment:
Oop, thought I had included this comment in my PR review. Let's add a test that incorporates `defaultValues` to [our tests](https://github.com/artsy/force/blob/a340dd41856f0ece085920053144836f50ad4116/src/v2/Components/ArtworkFilter/Utils/__tests__/isDefaultFilter.jest.tsx#L1) for this utility!